### PR TITLE
fix: require a filename for save as dialog

### PR DESCRIPTION
**Pull request by rmoesbergen**

### CODE REVIEW

File: src/lib/components/core/popups/popups/Prompt.svelte [hunk 1]
```diff
@@ -11,15 +11,23 @@ interface Props {
 	placeholder: string;
 	confirm: string;
 	value?: string;
+	requireValue: boolean;
```
Comment (sverben) [inline src/lib/components/core/popups/popups/Prompt.svelte:14]: this should probably be an optional parameter by adding a ? after it's name

**Review by sverben — CHANGES_REQUESTED** [review]
you can add the required prop on the input element instead of checking in the save method,  this will in most browsers also trigger a tooltip indicating you haven't filled out the field, providing additional user feedback

**Comment by rmoesbergen** [comment]
> you can add the required prop on the input element instead of checking in the save method, this will in most browsers also trigger a tooltip indicating you haven't filled out the field, providing additional user feedback

done, still need the return in save() to prevent the dialog from closing when the form is not valid

**Comment by koen1711** [comment]
Leaving review to @sverben

**Review by sverben — APPROVED** [review]

**Pull request merged**